ktor-server/ktor-server-core/jvm/src/io/ktor/server/engine/ShutdownHookJvm.kt

```diff
@@ -16,12 +17,12 @@ private val SHUTDOWN_HOOK_DISABLED = System.getProperty("io.ktor.server.engine.S
  * is already stopped then there will be no hook and no [stop] function invocation possible.
```

Comment:
The params are missing in the KDocs.

Also, I would rewrite the description:
"Configures automatic management of JVM shutdown hooks for terminating an application.
This function should be invoked **before** starting the application engine.
Once the JVM termination is detected, the [stop] block will be executed.
Please note that a shutdown hook is registered only while the application is running. If the application has already stopped, the hook won't be registered, and invoking [stop] will have no effect.
Therefore, the [stop] block will be called either once or not at all."